modular_zubbers/modules/modular_weapons/code/company_and_or_faction_based/carwo_defense_systems/ammo/rifle.dm

```diff
@@ -0,0 +1,11 @@
+/obj/projectile/bullet/c40sol
+	damage = 28
+	wound_bonus = -10
+	bare_wound_bonus = 0
+
+/obj/projectile/bullet/c40sol/pierce
+	damage = 20
+	armour_penetration = 25
+
+/obj/projectile/bullet/c40sol/incendiary
+	damage = 20
```

Comment:
```suggestion
	damage = 18
```
The damage should go lower since the main upside is that you're firing incendiary a someone.